plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSessionProperties.java

```diff
@@ -219,6 +222,11 @@ public IcebergSessionProperties(
                         "Target maximum size of written files; the actual size may be larger",
                         hiveConfig.getTargetMaxFileSize(),
                         false))
+                .add(stringProperty(
+                        HIVE_CATALOG_NAME,
+                        "Catalog to redirect to when a Hive table is referenced",
+                        icebergConfig.getHiveCatalogName().orElse(null),
+                        true))
```

Comment:
The hive one has a comment explaining why it's hidden. Want to copy it in?

```
                        // Session-level redirections configuration does not work well with views, as view body is analyzed in context
                        // of a session with properties stripped off. Thus, this property is more of a test-only, or at most POC usefulness.
```

Reply:
Thanks for the hint. I have missed this comment in the initial implementation.